docker-compose.accounts.yml

```diff
@@ -0,0 +1,163 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+services:
+  webapp:
+    build:
+      args:
+        WITH_ACCOUNTS: 1
+
+  nginx:
+    volumes:
+      - ./docker/accounts/nginx.secure.conf:/etc/nginx/conf.extra.d/nginx.secure.conf:ro
+
+  accounts:
+    build:
+      context: ./docker/accounts
+      dockerfile: Dockerfile
+    container_name: accounts
+    restart: unless-stopped
+    logging: *default-logging
+    env_file:
+      - .env
+    environment:
+      - SKYNET_DB_HOST=mongo
+      - SKYNET_DB_PORT=27017
+      - SKYNET_DB_USER=${SKYNET_DB_USER}
+      - SKYNET_DB_PASS=${SKYNET_DB_PASS}
+      - COOKIE_DOMAIN=${COOKIE_DOMAIN}
+      - COOKIE_HASH_KEY=${COOKIE_HASH_KEY}
+      - COOKIE_ENC_KEY=${COOKIE_ENC_KEY}
+    expose:
+      - 3000
+    networks:
+      shared:
+        ipv4_address: 10.10.10.70
+    depends_on:
+      - mongo
+      - oathkeeper
+
+  mongo:
+    image: mongo:4.4.1
+    command: --keyFile=/data/mgkey --replSet=skynet
+    container_name: mongo
+    restart: unless-stopped
+    logging: *default-logging
+    volumes:
+      - ./docker/data/mongo/db:/data/db
+      - ./docker/data/mongo/mgkey:/data/mgkey:rw
+    networks:
+      shared:
+        ipv4_address: 10.10.10.71
+    ports:
+      - "27017:27017"
+
+  kratos-migrate:
+    image: oryd/kratos:v0.5.4-alpha.1
+    container_name: kratos-migrate
+    restart: on-failure
+    logging: *default-logging
+    environment:
+      - DSN=cockroach://root@cockroach:26257/defaultdb?max_conns=20&max_idle_conns=4&sslmode=verify-full&sslcert=/certs/node.crt&sslkey=/certs/node.key&sslrootcert=/certs/ca.crt
+      - SQA_OPT_OUT=true
+    volumes:
+      - ./docker/kratos/config:/etc/config/kratos
+      - ./docker/data/cockroach/sqlite:/var/lib/sqlite
+      - ./docker/kratos/cr_certs:/certs
+    command: -c /etc/config/kratos/kratos.yml migrate sql -e --yes
+    networks:
+      shared:
+        ipv4_address: 10.10.10.80
+
+  kratos:
+    image: oryd/kratos:v0.5.4-alpha.1
+    container_name: kratos
+    restart: unless-stopped
+    logging: *default-logging
+    ports:
+      - "4433:4433" # public
+      - "4434:4434" # admin
+    expose:
+      - 4433 # public
+      - 4434 # admin
```

Comment:
I guess we don't need both but I'm not sure which to keep. I guess we're only going to be accessing this locally, so the `expose` makes more sense, right?